goodtables/inspector.py

```diff
@@ -29,26 +23,24 @@ class Inspector(object):
     # Public
 
     def __init__(self,
-                 checks='all',
-                 error_limit=1000,
-                 table_limit=10,
-                 row_limit=1000,
+                 checks='spec',
```

Comment:
Yes it's internal default. Also in readme Data Quality Spec is having an important place so I suppose user should understand what's going on calling `validate('data.csv')` (do all available **spec** checks).